### PR TITLE
PR: Fix About dialog format on macOS

### LINKED ISSUE #9298: About dialog on OSX is too big and bolded
The About dialog on mac is looking quite ugly (added the Mac about for comparison)

Screen Shot 2019-05-08 at 14 56 40

I will try to improve it

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

- [x] Reduce the size of text in OSX for about dialog
- [x] Enable url open in browser for General Preferences High DPI option

## Before
Screen Shot 2019-05-08 at 14 56 40

## After
Screen Shot 2019-05-09 at 01 41 16

## Linux

Screen Shot 2019-05-10 at 10 30 56

## Windows

Screen Shot 2019-05-10 at 10 45 49

### Issue(s) Resolved

Fixes #9298 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

**Comment by goanpeca** [comment]
This uses normal font (not bold) and reduces the text for OSX. The html was improved as well.

This is looking much better already, but since the `QMessageBox` has a fixed size, I think it would be better to make a dedicated `QAboutDialog(QDialog)` to be able to set a larger width in the future.

File: spyder/app/mainwindow.py [hunk 1]
```diff
@@ -2500,41 +2501,63 @@ def about(self):
...
         msgBox.setText(
             """
+            <div style='font-family: "{font_family}";
+                        font-size: {font_size}pt;
+                        font-weight: normal;
+                        '>
+            <p>
             <b>Spyder {spyder_ver}</b> {revision}
-            <br>The Scientific Python Development Environment |
+            The Scientific Python Development Environment |
```
Comment (CAM-Gerlach) [inline spyder/app/mainwindow.py]: You accidently removed an important ``<br>`` here

**Comment by CAM-Gerlach** [comment]
@goanpeca I'll review it shortly and ensure it still looks good on Windows (I can check Linux as well if @ccordoba12 doesn't). It definitely looks better in terms of the font size and face.

A few points right off the bat:

* You accidentally removed the ``<br>``s in the top block, so the alignment is all off, as you can see in the screenshot
* You can probably remove the old para about "Spyder is part of a larger effort" since that was added a very long time ago and really isn't necessary in the About dialog.
* There is no conceptual reason why our "Google Group" should be given more prominence than the other contact means by setting it off in its own para, in fact the opposite is true. It also takes up an extra line or more that way. Therefore, the ``<br><br>`` there should be removed. Similarly, the "many thanks" sentence should be part of the block above, otherwise it takes up extra space it doesn't need to, and the "distributed under" line as well, really.

**Comment by ccordoba12** [comment]
> You can probably remove the old para about "Spyder is part of a larger effort" since that was added a very long time ago and really isn't necessary in the About dialog.

I don't agree with that. Let's keep this PR in its right scope: it's only about the style of "About Spyder", not its contents.

**Comment by goanpeca** [comment]
@ccordoba12 updated the screenshots for kubuntu and windows. Besides a minor additional break this PR should only have affected OSX.

Goot to go?

File: spyder/preferences/general.py [hunk 2]
```diff
@@ -199,6 +199,7 @@ def setup_page(self):
                                            "these options (in "
                                            "English).").format(HDPI_QT_PAGE))
         screen_resolution_label.setWordWrap(True)
+        screen_resolution_label.setOpenExternalLinks(True)
```
Comment (goanpeca) [inline spyder/preferences/general.py]: This is a small fix for this dialog> I figured it was small enough to go here

Screen Shot 2019-05-10 at 10 49 15

**Comment by CAM-Gerlach** [comment]
@goanpeca Doing a final review now...

**Comment by goanpeca** [comment]
Which br @CAM-Gerlach ?

**Comment by CAM-Gerlach** [comment]
@goanpeca All of them, including the one I originally commented on.

File: spyder/app/mainwindow.py [hunk 3]
```diff
@@ -2500,41 +2501,59 @@ def about(self):
...
+            <div style='font-family: "{font_family}";
+                        font-size: {font_size}pt;
+                        font-weight: normal;
+                        '>
+            <p>
             <b>Spyder {spyder_ver}</b> {revision}
-            <br>The Scientific Python Development Environment |
+            </p>
+            <p>
+            The Scientific Python Development Environment |
```
Comment (CAM-Gerlach) [inline spyder/app/mainwindow.py]: This one

File: spyder/app/mainwindow.py [hunk 4]
```diff
@@ -2500,41 +2501,59 @@ def about(self):
...
             <a href="{website_url}">Spyder-IDE.org</a>
-            <br>Copyright &copy; 2009-2019 Spyder Project Contributors and
-            <a href="{github_url}/blob/master/AUTHORS.txt">others</a>
-            <br>Distributed under the terms of the
+            Copyright &copy; 2009-2019 Spyder Project Contributors and
```
Comment (CAM-Gerlach) [inline spyder/app/mainwindow.py]: This one

File: spyder/app/mainwindow.py [hunk 5]
```diff
@@ -2500,41 +2501,59 @@ def about(self):
...
+            <a href="{github_url}/blob/master/AUTHORS.txt">others</a>.
+            Distributed under the terms of the
```
Comment (CAM-Gerlach) [inline spyder/app/mainwindow.py]: This one

**Comment by CAM-Gerlach** [comment]
@goanpeca I made a comment explicitly highlighting each one for you.

**Comment by goanpeca** [comment]
Screen Shot 2019-05-10 at 23 14 47

**Comment by goanpeca** [comment]
Will do

**Comment by ccordoba12** [comment]
> If you could submit another PR to fix the regression introduced by this one, I would greatly appreciate it. Thanks.

This is not good. From now on PRs will need to be approved to be able to be merged by core developers (except me because sometimes I need to merge things quickly).

**Comment by CAM-Gerlach** [comment]
In particular, even if at least one approval isn't required (as one should be, I agree, and I thought that rule was already enforced on this repo), I assumed merging was blocked for non-admins (i.e. you) if a non-dismissed, changes-required (red) review was present on this PR (and I specifically set it to that to ensure it wasn't merged until the ``<br>`` issue was fixed). Clearly, that's not the case.

**Comment by goanpeca** [comment]
Yes guys I am sorry I really did not understand what the problem was and I thought I fixed it. To be honest the oversize and bold font was upsetting me way more than those line breaks on OSX 😬

**Comment by ccordoba12** [comment]
@CAM-Gerlach, sorry if I was a bit blunt with my message. I was not saying that your comment was wrong, I was just informing you and @goanpeca about our new policy on how to merge PRs.

> In particular, even if at least one approval isn't required (as one should be, I agree, and I thought that rule was already enforced on this repo)

Nop, it wasn't. I just activated it to avoid this situation in the future, i.e. merging your own PRs without waiting for the approval of at least another developer.

> I assumed merging was blocked for non-admins

Nop, all members of `@spyder-ide/core-developers` (the one with write permissions to this repo) can still merge PRs.

**Comment by CAM-Gerlach** [comment]
> sorry if I was a bit blunt with my message. 

Thanks, but I actually didn't have a problem with it, I was actually agreeing with you :) I was probably a bit too blunt with mine, to @goanpeca , since we all make mistakes (which is what the approval policy while hopefully help us help each other avoid)

> Nop, it wasn't. I just activated it to avoid this situation in the future,

Great! :+1: 

> (the ones with write permissions to this repo) can still merge PRs.

Yeah, I'm aware of that part (I'm one of them :); the key part of the quoted phrase was

> **if a non-dismissed, changes-required (red) review was present**

but evidently that's not how it works and Github only considers approvals, not disapprovals.

**Comment by ccordoba12** [comment]
> but evidently that's not how it works and Github only considers approvals, not disapprovals.

Yep, I think that's right.

**Pull request merged**